### DISCUSSION TITLE
Delay Merge -> API Update

### DISCUSSION THREAD

**Discussion by ChristianKraemer** [Homebrew/discussions#4583] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.24-9-gf0be2ca
ORIGIN: https://github.com/Homebrew/brew
HEAD: f0be2ca4d5ee61f5a7eda5a51bbf2b92cab9e1fc
Last commit: 33 minutes ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 29d8cdcc79a612cd044a7481b6834c1007afaeb4
Core tap last commit: 17 minutes ago
Core tap branch: master
Core tap JSON: 20 Jun 15:45 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.Y0dOcBgpDe/org.xquartz:0
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 4
HOMEBREW_NO_ENV_HINTS: set
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: quad-core 64-bit kabylake
Clang: 14.0.3 build 1403
Git: 2.41.0 => /usr/local/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4-x86_64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Firefox offers an update from 114.0.1 to 114.0.2.
I see, that 2 hours ago a PR regarding that update was merged. But neither "brew upgrade --greedy-auto-updates" nor "brew reinstall firefox" updates/installs the firefox.
From my point of view two  hours delay is way to much.

**Comment by Bo98** [comment, marked as answer]
Thanks for flagging this issue. The API is usually regenerated every 15 minutes, but I've found an issue that caused it to stop working for the last few hours. It should be up and running again shortly, and I will look into the root cause to try and prevent it reoccurring.

**Reply by Bo98** [reply]
It appears to be an issue on GitHub's side but we're working on getting it resolved as soon as possible.
